### PR TITLE
Hyo Tester's First Focused Reflection

### PR DESCRIPTION
**Pull request by hyo9292**
Hi, I'm still working on the focused reflection and will complete by the end of today!

### CODE REVIEW

**Comment by haley-triem** [comment]
Makes sense! Best of luck on your focused reflection!

**Review by haley-triem — CHANGES_REQUESTED** [review]
Leaving a comment to practice the "request changes" function; your file name looks good, and it seems like all you have to do is add in the function!

**Review by haley-triem — CHANGES_REQUESTED** [review]
Testing one more comment to see if I am doing this review thing right!!

**Review by hyo9292 — COMMENTED** [review]
I update few codes to share the loop/ variable function!

**Review by haley-triem — COMMENTED** [review]
Hyo, I love your birthday code and reflection! I don't want to approve your code until I am sure that it is done because I don't want to accidentally post it to the website if you are still wanting to make changes ... would you like me to do a final check and approve your pull request, or are you still making changes? Thank you!

**Comment by hyo9292** [comment]
Hey, Haley!! I'm not going to make any changes! Could you please check and confirm the codes to be approved for my pull request?  thanks!

**Comment by haley-triem** [comment]
Hi Hyo, sounds good! I will take a look at your code and approve the request.

File: _posts/hyotester/2022-09-22-hyotester-first-focused-reflection.md [hunk 1]
```diff
@@ -1,8 +1,202 @@
...
+from datetime import datetime
+today = datetime.today()
+age_str = input('What is your age?\n')
+age = int(age_str)
+birth_year = today.year - age
+print("You were born in " + str(birth_year) + " or " + str(birth_year - 1))
+
+```
+
+#Creating a Cake with Topping and a Plate
```
Comment (haley-triem) [inline _posts/hyotester/2022-09-22-hyotester-first-focused-reflection.md:55]: Hi Hyo, was this meant to be a header for the next section? I think you might have forgotten a space between "#" and "Creating a Cake with Topping and a Plate"

File: _posts/hyotester/2022-09-22-hyotester-first-focused-reflection.md [hunk 2]
```diff
@@ -1,8 +1,202 @@
...
+
+#The loop / variable name 
+
+```python
+for name in ["Joe", "Amy", "Brad", "Angelina", "Zuki", "Thandi", "Paris"]:
+    print("Hi", name, "Please come to my party on Saturday!")
+        
+```
+
+# Input
```
Comment (haley-triem) [inline _posts/hyotester/2022-09-22-hyotester-first-focused-reflection.md:26]: I might rename this to something like "User Input to Calculate Birth Year and Number of Friends" ... I think your code is really cool, but I think a more specific title would clarify what exactly your code is doing (this goes with what prof. Elliott was saying about writing for the humans reading the code) ... what do you think?

**Comment by haley-triem** [comment]
Dear Hyo -- I left two suggestions after reading through your code; please let me know your thoughts on whether or not you want to take those suggestions (absolutely no worries if you don't want to take them!), and I will be sure to merge your pull request after! Warm regards, Haley

File: _posts/hyotester/2022-09-22-hyotester-first-focused-reflection.md (hunk 2 above)
Comment (hyo9292) [reply _posts/hyotester/2022-09-22-hyotester-first-focused-reflection.md:26]: Yes, exactly I was looking for a more specific title, and "User Input to Calculate Birth Year and Number of Friends" make sense to me! Thank you for the suggestion!

**Comment by hyo9292** [comment]
Dear Haley!! Thank you for two suggestions! And I fixed the codes. Thank your comments! Appreciate your time! Yes, could you please merge my request? And see you in the class!

**Review by haley-triem — APPROVED** [review]
I think that your reflection is a little more clear now; I like your cake! I am going to merge the pull request!

**Pull request merged**